bundles/org.eclipse.rap.rwt/src/org/eclipse/rap/rwt/internal/textsize/TextSizeStorageUtil.java

```diff
@@ -65,6 +69,13 @@ static Integer getKey( FontData fontData, String string, int wrapWidth, int mode
     return Integer.valueOf( hashCode );
   }
 
+  static TextSizeStorage getTextSizeStorage() {
+    if(RWTProperties.isTextSizeStoreSessionScoped()) {
```

Comment:
To avoid performance problems by querying a system property too often, please keep it in a static final boolean field.